evap/static/scss/components/_buttons.scss

```diff
@@ -192,6 +192,18 @@ a:not([href]):not(.disabled) {
         width: 5rem;
         font-size: 0.8rem;
     }
+
+    &.focus, &:focus {
+        outline: none;
+        box-shadow: 0 0 0 0.2rem rgba(49, 112, 143, 0.5);
```

Comment:
Should this be a constant?

Reply:
(also used a couple of lines further down)

Reply:
the shadow doesn't need to be a constant i think, but please use `$evap-dark-blue` for the rgb part.